guides-markdown/events.md

```diff
@@ -0,0 +1,115 @@
+| `tx` | [TX](https://github.com/bcoin-org/bcoin/blob/master/lib/primitives/tx.js)| [Pool](https://github.com/bcoin-org/bcoin/blob/master/lib/net/pool.js): `_handleTX()`<br>(only if there is no mempool, i.e. SPV) | _pool_&#8594;[SPVNode](https://github.com/bcoin-org/bcoin/blob/master/lib/node/spvnode.js) |
+| `tx` | [TX](https://github.com/bcoin-org/bcoin/blob/master/lib/primitives/tx.js), [Details](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/txdb.js) | [TXDB](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/txdb.js): `insert()` | _txdb_&#8594;[WalletDB](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/walletdb.js) (also&nbsp;returns&nbsp;Wallet)<br>_txdb_&#8594;[Wallet](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/wallet.js)|
+| `double spend` | [MempoolEntry](https://github.com/bcoin-org/bcoin/blob/master/lib/mempool/mempoolentry.js) | [Mempool](https://github.com/bcoin-org/bcoin/blob/master/lib/mempool/mempool.js): `removeDoubleSpends()` | |
+| `balance` | [Balance](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/txdb.js) | [TXDB](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/txdb.js): `insert()`, `confirm()`, `disconnect()`, `erase()` | _txdb_&#8594;[WalletDB](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/walletdb.js) (also&nbsp;returns&nbsp;Wallet)<br>_txdb_&#8594;[Wallet](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/wallet.js)|
+| `address` |  [[WalletKey](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/walletkey.js)] | [Wallet](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/wallet.js): `_add()`,  |  _wallet_&#8594;[WalletDB](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/walletdb.js) (returns&nbsp;parent&nbsp;Wallet,&nbsp;[WalletKey]) |
+
+## Socket Events
+
+Websocket connections in bcoin are handled by two servers, one for `Node` and one for `Wallet`.
+Those servers each have child objects such as `Chain`, `Mempool`, `Pool`, and `WalletDB`, and
+relay events from them out the socket. To receive an event, the socket client must watch a
+channel (such as `chain`, `mempool`, or `auth`) or join the a wallet
+(which would be user-defined like `primary`, `hot-wallet`, or `multisig1`). All wallets can be
+joined at once by joining `'*'`. See more about joining and watching with hooks and calls below.
+
+### Wallet
+
+All wallet events are triggered by a `WalletDB` object, which have may be triggered by its parent
+`TXDB` or `Wallet`. The socket emits the event along with the wallet ID, and whatever "Returns" are listed above.
+
+| Event |
```

Comment:
Even if it's repetitive, might still be helpful to have the returns data in the table here too, especially since it's included for node below. Even if it's the same, I feel it helps with scanning this document